glass/inference/glass_runner.py

```diff
@@ -1,42 +1,35 @@
+import os
+import cv2
+import torch
 import logging
-from timeit import default_timer as timer
-from typing import List
-
 import numpy as np
-import torch
+from typing import List
 import torch.nn.functional as F
-
-from detectron2.checkpoint import DetectionCheckpointer
 from detectron2.config import get_cfg
 from detectron2.modeling import build_model
-from detectron2.structures import Instances
-
-from glass.config import add_e2e_config, add_glass_config, add_dataset_config, add_post_process_config
-from glass.postprocess import build_post_processor
 from glass.utils.common_utils import rgb2grey
+from glass.postprocess import build_post_processor
+from detectron2.checkpoint import DetectionCheckpointer
+from detectron2.structures import Instances, RotatedBoxes
 from glass.modeling.recognition.text_encoder import TextEncoder
+from Trax.Miscellaneous.Users.OriS.OCR.glass_det.glass_main.glass.utils.visualizer import visualize
```

Comment:
Are you sure you want to import something for personal folder? maybe you should move it Trax.Algo.External under name GlassOCR

Reply:
Good catch, I thought I already removed it.